aten/src/ATen/native/cuda/UnaryGeometricAcosKernel.cu

```diff
@@ -5,19 +5,21 @@
 #include <ATen/native/DispatchStub.h>
 #include <ATen/native/TensorIterator.h>
 #include <ATen/native/UnaryOps.h>
+#include <ATen/native/cuda/JitLoops.cuh>
 #include <ATen/native/cuda/Loops.cuh>
 #include <ATen/native/cuda/Math.cuh>
 #include <limits>
 
 namespace at::native {
 
-CONSTEXPR_EXCEPT_WIN_CUDA char acos_name[] = "acos";
+CONSTEXPR_EXCEPT_WIN_CUDA char acos_name[] = "acos_impl";
 void acos_kernel_cuda(TensorIteratorBase& iter) {
   auto common_dtype = iter.common_dtype();
   if (at::isComplexType(common_dtype)) {
-#if AT_USE_JITERATOR
+    // Disabled due to accuracy issues
```

Comment:
Some of these might just need tolerance bumps, but I didn't want to spend much time on this.

Reply:
Could you open an issue describing what would need doing and tag it as "good first issue"?

Reply:
Opened #100842